CHANGELOG.md

```diff
@@ -96,7 +104,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 - `ChargeSubcategory` column - See `ChargeCategory` and `ChargeClass` columns
 
-[All unreleased changes](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/compare/v1.0-preview-cr...working_draft)
+[All 1.0 changes](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/compare/v1.0-preview-cr...1.0-cr)
```

Comment:
Should this `diff` be established once the `v1.0` and `v1.0-preview` are published? In this case, the `-cr` qualification is not needed. At the publication, we will create a Release tag for `v1.0` and `v1.0-preview`.